README.md

```diff
@@ -2482,6 +2489,149 @@ Client.newHazelcastClient().then(function (client) {
+    ]
+}
+```
+
+**Programmatic:**
+
+```javascript
+var nearCacheConfig = new Config.NearCacheConfig();
+nearCacheConfig.name = 'myDataStructure';
+nearCacheConfig.invalidateOnChange = (false|true);
+nearCacheConfig.timeToLiveSeconds = (0..Number.MAX_SAFE_INTEGER);
+nearCacheConfig.maxIdleSeconds = (0..Number.MAX_SAFE_INTEGER);
+nearCacheConfig.inMemoryFormat= (InMemoryFormat.OBJECT|InMemoryFormat.BINARY);
+nearCacheConfig.evictionPolicy = (EvictionPolicy.LRU|EvictionPolicy.LFU|EvictionPolicy.RANDOM|EvictionPolicy.NONE);
+nearCacheConfig.evictionMaxSize = (0..Number.MAX_SAFE_INTEGER);
+nearCacheConfig.evictionSamplingCount = (0..Number.MAX_SAFE_INTEGER);
+nearCacheConfig.evictionSamplingPoolSize = (0..Number.MAX_SAFE_INTEGER);
+
+cfg.nearCacheConfigs['mostlyReadMap'] = nearCacheConfig;
+```
+
+Following are the descriptions of all configuration elements:
+
+- `inMemoryFormat`: Specifies in which format data will be stored in your Near Cache. Note that a map’s in-memory format can be different from that of its Near Cache. Available values are as follows:
+  - `BINARY`: Data will be stored in serialized binary format (default value).
+  - `OBJECT`: Data will be stored in deserialized form.
+
+- `invalidateOnChange`: Specifies whether the cached entries are evicted when the entries are updated or removed in members. Its default value is true.
+
+- `timeToLiveSeconds`: Maximum number of seconds for each entry to stay in the Near Cache. Entries that are older than this period are automatically evicted from the Near Cache. Regardless of the eviction policy used, `timeToLiveSeconds` still applies. Any integer between 0 and `Number.MAX_SAFE_INTEGER`. 0 means infinite. Its default value is 0.
+
+- `maxIdleSeconds`: Maximum number of seconds each entry can stay in the Near Cache as untouched (not read). Entries that are not read more than this period are removed from the Near Cache. Any integer between 0 and `Number.MAX_SAFE_INTEGER`. 0 means `Number.MAX_SAFE_INTEGER`. Its default value is 0. 0 means infinite.
+
+- `evictionPolicy`: Eviction policy configuration. Available values are as follows:
+  - `LRU`: Least Recently Used (default value).
+  - `LFU`: Least Frequently Used.
+  - `NONE`: No items are evicted and the `evictionMaxSize` property is ignored. You still can combine it with `timeToLiveSeconds` and `maxIdleSeconds` to evict items from the Near Cache. 
+  - `RANDOM`: A random item is evicted.
+  
+- `evictionMaxSize`: Maximum number of entries kept in the memory before eviction kicks in.
+- `evictionSamplingCount`: Number of random entries that are evaluated to see if some of them are already expired. If there are expired entries, those are removed and there is no need for eviction.
+- `evictionSamplingPoolSize`: Size of the pool for eviction candidates. The pool is kept sorted according to eviction policy. The entry with the highest score is evicted. 
+
+#### 7.8.2.2. Near Cache Example for Map
+
+Let's explain the below configuration.
```

Comment:
I meant, maybe we could explain what the below configuration's behavior is :) Something similar to: "The following is an example configuration for a Near Cache defined in the `mostlyReadMap` map. According to this configuration, the entries are stored as `object`'s in this Near Cache and eviction starts when the count of entries reaches `5000`; entries are evicted based on the `lru` (Least Recently Used) policy. In addition, when an entry is updated or removed on the member side, it is immediately evicted on the client side."